cmd/subctl/join.go

```diff
@@ -38,8 +38,9 @@ import (
 )
 
 var (
-	joinFlags    join.Options
-	labelGateway bool
+	joinFlags      join.Options
+	ignoredIkePort int
```

Comment:
Should we add a todo comment here to remove it?

Reply:
The [recommended way](https://go.dev/blog/godoc) was is to add a doc comment that begins with “Deprecated:”